bin/tictactoe

```diff
@@ -1,3 +1,55 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+def top
+  puts "Welcome to Tic-Tac-Toe (Object-Oriented Version)"
+  puts "======================"
+end
+
+def pick_setup
+  x = 3
```

Comment:
It would be preferable to set the variable `x` to be named something more appropriate like `input` and it is much better to set `input = nil`